preprocessor/folder/folder.go

```diff
@@ -46,17 +61,24 @@ func Run(args []string) error {
 		return err
 	}
 
-	// Scan whole sync directory
-	fileList, err := scanSyncDirectory("")
+	projects, err := queryAllProjects(pgPool)
 	if err != nil {
 		return err
 	}
-	for _, filePath := range fileList {
-		cid, err := handleNewFile(pgPool, filePath)
+
+	for _, project := range projects {
+		projectPath := project.ProjectPath
+		fileList, err := scanSyncDirectory(projectPath)
 		if err != nil {
 			log.Println(err)
-		} else {
-			log.Printf("file %s uploaded to webhook with CID %s\n", filePath, cid)
+		}
+		for _, filePath := range fileList {
+			cid, err := handleNewFile(pgPool, filePath, &project)
+			if err != nil {
+				log.Println(err)
+			} else if cid != "" {
+				log.Printf("File %s uploaded to webhook with CID %s\n", filePath, cid)
```

Comment:
```suggestion
				log.Printf("file %s uploaded to webhook with CID %s", filePath, cid)
```

The logger will add newlines to the output if needed.